File: MVC/Spartane.Web/Uploads/Scripts/Spartan_Metadata.js
$(document).ready(function () {
    var inpuElementArray = [
        { "inputId": "AttributeId", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "Default", "HelpText": "" }
        ,{ "inputId": "Class_Id", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Class_Name", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Object_Id", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Physical_Name", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Logical_Name", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Identifier_Type", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Attribute_Type", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Length", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Decimals_Length", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Relation_Type", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Related_Object_Id", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Related_Class_Id", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Related_Class_Name", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Related_Class_Identifier", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Related_Class_Description", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Required", "inputType": "checkbox", "IsRequired": false, "IsVisible": true, "IsReadOnly": false, "DefaultValue": null, "HelpText": "" }
        ,{ "inputId": "DefaultValue", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Visible", "inputType": "checkbox", "IsRequired": false, "IsVisible": true, "IsReadOnly": false, "DefaultValue": null, "HelpText": "" }
        ,{ "inputId": "Help_Text", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Read_Only", "inputType": "checkbox", "IsRequired": false, "IsVisible": true, "IsReadOnly": false, "DefaultValue": null, "HelpText": "" }
        ,{ "inputId": "ScreenOrder", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Control_Type", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Group_Name", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }

    ];
    setInputEntityAttributes(inpuElementArray, "#", 1);
});

File: MVC/Spartane.Web/Resource/GetResource/Spartan_Metadata.cs
using Resources.Abstract;
using Resources.Concrete;
using System;
using System.Globalization;
using System.Configuration;
using System.IO;

namespace Resources
{
    public partial class Spartan_MetadataResources
    {
        //private static IResourceProvider resourceProviderSpartan_Metadata = new XmlResourceProvider(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Uploads\Resources\Spartan_MetadataResource." + CultureInfo.CurrentUICulture.Name + ".xml"));
        private static IResourceProvider resourceProviderSpartan_Metadata = new XmlResourceProvider(Path.Combine(ConfigurationManager.AppSettings["BaseDirectoyPhysical"].ToString(), @"Uploads\Resources\Spartan_MetadataResource." + CultureInfo.CurrentUICulture.Name + ".xml"));

        /// <summary>Spartan_Metadata</summary>
        public static string Spartan_Metadata
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Spartan_Metadata", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>AttributeId</summary>
        public static string AttributeId
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("AttributeId", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Class_Id</summary>
        public static string Class_Id
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Class_Id", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Class_Name</summary>
        public static string Class_Name
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Class_Name", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Object_Id</summary>
        public static string Object_Id
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Object_Id", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Physical_Name</summary>
        public static string Physical_Name
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Physical_Name", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Logical_Name</summary>
        public static string Logical_Name
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Logical_Name", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Identifier_Type</summary>
        public static string Identifier_Type
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Identifier_Type", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Attribute_Type</summary>
        public static string Attribute_Type
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Attribute_Type", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Length</summary>
        public static string Length
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Length", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Decimals_Length</summary>
        public static string Decimals_Length
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Decimals_Length", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Relation_Type</summary>
        public static string Relation_Type
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Relation_Type", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Related_Object_Id</summary>
        public static string Related_Object_Id
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Related_Object_Id", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Related_Class_Id</summary>
        public static string Related_Class_Id
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Related_Class_Id", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Related_Class_Name</summary>
        public static string Related_Class_Name
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Related_Class_Name", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Related_Class_Identifier</summary>
        public static string Related_Class_Identifier
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Related_Class_Identifier", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Related_Class_Description</summary>
        public static string Related_Class_Description
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Related_Class_Description", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Required</summary>
        public static string Required
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Required", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>DefaultValue</summary>
        public static string DefaultValue
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("DefaultValue", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Visible</summary>
        public static string Visible
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Visible", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Help_Text</summary>
        public static string Help_Text
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Help_Text", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Read_Only</summary>
        public static string Read_Only
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Read_Only", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>ScreenOrder</summary>
        public static string ScreenOrder
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("ScreenOrder", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Control_Type</summary>
        public static string Control_Type
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Control_Type", CultureInfo.CurrentUICulture.Name) as String;
            }
        }

        /// <summary>Group_Name</summary>
        public static string Group_Name
        {
            get
            {
                return resourceProviderSpartan_Metadata.GetResource("Group_Name", CultureInfo.CurrentUICulture.Name) as String;
            }
        }


    }
}
